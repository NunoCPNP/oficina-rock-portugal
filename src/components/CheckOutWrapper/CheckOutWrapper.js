import useTranslation from 'next-translate/useTranslation'

import SEO from '@/components/SEO'
import SectionTitle from '@/components/SectionTitle'

const CheckOutWrapper = ({ children }) => {
  const { t } = useTranslation('common')

  return (
    <>
      <SEO title="Oficina Rock Portugal" description="" />
      <SectionTitle title={t(`checkout`)} offset="4rem" />
      {children}
    </>
  )
}

export default CheckOutWrapper
