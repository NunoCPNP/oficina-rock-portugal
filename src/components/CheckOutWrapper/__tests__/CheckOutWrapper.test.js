import { render } from '@testing-library/react'

import CheckOutWrapper from '..//CheckOutWrapper'

describe('<CheckOutWrapper /> spec', () => {
  it('Should match snapshot', () => {
    const { container } = render(<CheckOutWrapper />)

    expect(container).toMatchSnapshot()
  })
})
