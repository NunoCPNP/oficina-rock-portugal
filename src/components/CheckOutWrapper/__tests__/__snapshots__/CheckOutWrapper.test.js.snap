// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckOutWrapper /> spec Should match snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  text-transform: uppercase;
  margin-top: 4rem;
  margin-bottom: 4rem;
}

.emotion-0 h3 {
  padding-left: 0.5rem;
}

@media only screen and (max-width: 480px) {
  .emotion-0 {
    margin-bottom: 2rem;
  }
}

<div>
  <div
    class="emotion-0"
    offset="4rem"
  >
    <h2>
      common:checkout
    </h2>
  </div>
</div>
`;
